src/lib/api.ts

```diff
@@ -30,15 +30,14 @@ export function enableGlobalGraphQLErrorHandling(): void {
 	globalGraphQLErrorHandlingEnabled = true;
 }
 
-export default async function API( { exitOnError = true } = {} ): Promise<
-	ApolloClient< NormalizedCacheObject >
-> {
-	const authToken = await Token.get();
-	const headers = {
-		'User-Agent': env.userAgent,
-		Authorization: `Bearer ${ authToken.raw }`,
-	};
-
+// eslint-disable-next-line @typescript-eslint/require-await
+export default async function API( {
```

Comment:
Now that we no longer await, we should remove `async`. Might require some cleanup where it is called

```suggestion
export default function API( {
```

Reply:
You caught me :< I was trying to get out of cleaning it up